js/views/pageNavVw.js

```diff
@@ -225,18 +224,16 @@ module.exports = baseVw.extend({
         username,
         avatar,
         avatarHash,
-        notif,
-        notifStamp;
+        notif;
 
     if (data.hasOwnProperty('notification') || data.hasOwnProperty('message') && data.message.subject) {
       notif = data.notification || data.message;
       username = notif.handle ? notif.handle : notif.guid;
       username = notif.sender ? notif.sender : username;
       avatarHash = notif.image_hash || notif.avatar_hash;
       avatar = avatarHash ? app.serverConfigs.getActive().getServerBaseUrl + '/get_image?hash=' +
-        avatarHash + '&guid=' + notif.guid : 'imgs/defaultUser.png';
```

Comment:
can we keep the leading spaces, since the second line is an extension of the first one?